integrationExamples/gpt/audigentSegments_example.html

```diff
@@ -232,17 +245,29 @@
                     auctionDelay: 1000
                 },
                 realTimeData: {
-                    auctionDelay: 1000,
-                    dataProviders: [{name: "audigent", waitForIt: true}]
+                    auctionDelay: testAuctionDelay, // lower in real scenario to meet publisher spec
+                    dataProviders: [
+                        {
+                            name: "audigent",
+                            waitForIt: true,
+                            params: {
+                                mapSegments: ['appnexus'],
```

Comment:
Is the value of mapSegments an array or an object? This example is inconsistent with the .md file below.

Reply:
It's an object, thanks for the catch.